### ISSUE TITLE
FatWatch source code project is too 'old'

### ISSUE THREAD

**Issue by swinghu** [heroic-software/FatWatch#2]
FatWatch is an awesome app, i guess it may be a very old app,the developer build many years ago.
Right now , I git clone the source code in my Xcode 7.2, it promt error message 

`/ios-dev/FatWatch/BRTables/BRTableRow.h:34:23: 'UITextAlignment' is deprecated: first deprecated in iOS 6.0`

deprecated

**Comment by benzado** [comment]
Yes, it is very old. I started writing it when the "iPhoneOS SDK" was in beta.

Unfortunately I wasn't able to keep it up to date.

At this point, the smart thing to do would be to write a new app, using the FatWatch code as a reference.

**Comment by swinghu** [comment]
At last, I run it successfully on my mac OS X Yosemite, Xcode 7.2

**Comment by augmentedfourth** [comment]
@swinghu Can you share what it took for you to get it running? I really, really miss this app.
